### PR TITLE
Rename group id and artifact id for Azure Monitor exporter

### PR DESCRIPTION
**Pull request by srnagar**
This PR renames the `opentelemetry-exporters-azuremonitor` artifact to `microsoft-opentelemetry-exporter-azuremonitor`. Additionally, this artifact will now be under `com.microsoft` Maven groupId.

### CODE REVIEW

File: sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ search for your resource. On the overview page of your resource, you will find t
 right corner.
 
 ### Creating exporter for Azure Monitor
-<!-- embedme ./src/samples/java/com/azure/opentelemetry/exporters/azuremonitor/ReadmeSamples.java#L26-L28 -->
+<!-- embedme ./src/samples/java/com/microsoft/opentelemetry/exporters/azuremonitor/ReadmeSamples.java#L26-L28 -->
```
Comment (trask) [inline sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md:41]:
```suggestion
<!-- embedme ./src/samples/java/com/microsoft/opentelemetry/exporter/azuremonitor/ReadmeSamples.java#L26-L28 -->
```

File: sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/pom.xml [hunk 2]
```diff
@@ -12,9 +12,9 @@
     <relativePath>../../parents/azure-client-sdk-parent</relativePath>
   </parent>
 
-  <groupId>com.azure</groupId>
-  <artifactId>opentelemetry-exporters-azuremonitor</artifactId>
-  <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:opentelemetry-exporters-azuremonitor;current} -->
+  <groupId>com.microsoft</groupId>
+  <artifactId>microsoft-opentelemetry-exporter-azuremonitor</artifactId>
+  <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:microsoft-opentelemetry-exporter-azuremonitor;current} -->
```
Comment (trask) [inline sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/pom.xml:17]:
```suggestion
  <version>1.0.0-beta.2</version> <!-- {x-version-update;com.microsoft:microsoft-opentelemetry-exporter-azuremonitor;current} -->
```

File: sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ AzureMonitorExporter azureMonitorExporter = new AzureMonitorExporterBuilder()
 The following example shows how to export a collection of available [Spans][span_data] to Azure Monitor through the
  `AzureMonitorExporter`
 
-<!-- embedme ./src/samples/java/com/azure/opentelemetry/exporters/azuremonitor/ReadmeSamples.java#L35-L40 -->
+<!-- embedme ./src/samples/java/com/microsoft/opentelemetry/exporters/azuremonitor/ReadmeSamples.java#L35-L40 -->
```
Comment (trask) [inline sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md:53]:
```suggestion
<!-- embedme ./src/samples/java/com/microsoft/opentelemetry/exporter/azuremonitor/ReadmeSamples.java#L35-L40 -->
```

File: eng/jacoco-test-coverage/pom.xml [hunk 4]
```diff
@@ -273,8 +273,8 @@
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>opentelemetry-exporters-azuremonitor</artifactId>
-      <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:opentelemetry-exporters-azuremonitor;current} -->
+      <artifactId>microsoft-opentelemetry-exporter-azuremonitor</artifactId>
+      <version>1.0.0-beta.2</version> <!-- {x-version-update;com.azure:microsoft-opentelemetry-exporter-azuremonitor;current} -->
```
Comment (trask) [inline eng/jacoco-test-coverage/pom.xml:277]:
```suggestion
      <version>1.0.0-beta.2</version> <!-- {x-version-update;com.microsoft:microsoft-opentelemetry-exporter-azuremonitor;current} -->
```

File: eng/jacoco-test-coverage/pom.xml [hunk 5]
```diff
@@ -273,8 +273,8 @@
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
```
Comment (trask) [inline eng/jacoco-test-coverage/pom.xml:275]:
```suggestion
      <groupId>com.microsoft</groupId>
```

File: eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml [hunk 6]
```diff
@@ -208,7 +208,7 @@
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
             files="com.azure.core.http.okhttp.OkHttpAsyncHttpClientBuilder"/>
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck"
-            files="com.azure.opentelemetry.exporters.azuremonitor.AzureMonitorExporter"/>
+            files="AzureMonitorExporter"/>
```
Comment (trask) [inline eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml:211]: is this correct with no package name here?

Comment (srnagar) [reply eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml]: Good catch! I think IDE refactoring removed the package name. Added the package name back.

**Review by joshfree — APPROVED** [review]

**Pull request merged**